README.md

```diff
@@ -61,8 +61,7 @@ flatpak remote-add --user --if-not-exists flathub https://flathub.org/repo/flath
 
 For KDE apps:
 ```
-curl -O https://distribute.kde.org/kdeapps.flatpakrepo
-flatpak remote-add --user --if-not-exists kdeapps kdeapps.flatpakrepo
+flatpak remote-add --user --if-not-exists kdeapps http://distribute.kde.org/kdeapps.flatpakrepo
```

Comment:
Does it work for you on Sailfish 3.3? For me it did not work when I was working on it.

Reply:
it works via `http` and flatpak repo url inside flatpakrepo file have http scheme anyway, so

Reply:
Thanks!